scss/civicrm/dashboard/pages/_edit-dashboard.scss

```diff
@@ -0,0 +1,92 @@
+#help {
```

Comment:
if all this styling is for just when the dashboard is in "edit mode", i believe there should be a selector wrapping this entire style no?